doc/how-to/recover.md

```diff
@@ -0,0 +1,63 @@
+# How to recover a MicroCloud cluster
+
+Each MicroCloud service uses the [Dqlite](https://dqlite.io/) distributed
+database for highly-available storage. While the cluster recovery process is
+similar for each service, this document only covers cluster recovery for the
+`microcloudd` daemon. For cluster recovery procedures for LXD, MicroCeph and
+MicroOVN, see:
+
+- [LXD Cluster Recovery](https://documentation.ubuntu.com/lxd/en/latest/howto/cluster_recover/)
+- [MicroOVN Launchpad Bug](https://bugs.launchpad.net/microovn/+bug/2072377)
+- [MicroCeph Issue](https://github.com/canonical/microceph/issues/380)
+
+If a MicroCloud cluster loses more than a quorum of its database members, then
```

Comment:
Would be nice to define what is the minimum qualification of "quorum"

Reply:
I was initially going to go into way too much detail here regarding quorum and member roles, but that stuff is really more in scope in the Dqlite docs, not here. That said, the dqlite docs [do not include](https://github.com/canonical/dqlite/issues/369) any information on member roles beyond what's present in the [wire protocol spec](https://dqlite.io/docs/reference/wire-protocol).
I rearranged this a bit and I think it improves clarity.